### PR TITLE
[zh] Translate tasks/tls/manual-rotation-of-ca-certificates.md

### PR DESCRIPTION
**Pull request by tengqm**
closes: #23171

### CODE REVIEW

**Review by markthink — COMMENTED** [review]
/lgtm
/approve

File: content/zh/docs/tasks/tls/manual-rotation-of-ca-certificates.md [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+      * 重启调度器以使用新的 CA 证书。
+      * 确保控制面组件的日志中没有 TLS 相关的错误信息。
+
+      <!--
+      To generate certificates and private keys for your cluster using the `openssl`
+      command line tool, see [Certificates (`openssl`)](/docs/concepts/cluster-administration/certificates/#openssl).
+      You can also use [`cfssl`](/docs/concepts/cluster-administration/certificates/#cfssl).
+      -->
+      {{< note >}}
+      要使用 `openssl` 命令行红菊为集群生成新的证书和私钥，可参阅
```
Comment (SataQiu) [inline content/zh/docs/tasks/tls/manual-rotation-of-ca-certificates.md:192]:
```suggestion
      要使用 `openssl` 命令行工具为集群生成新的证书和私钥，可参阅
```

File: content/zh/docs/tasks/tls/manual-rotation-of-ca-certificates.md [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+
+   1. Restart the control plane components by removing the old CA from the kubeconfig files and the files against `--client-ca-file`, `--root-ca-file` flags resp.
+
+   1. Restart kubelet by removing the old CA from file against the `clientCAFile` flag and kubelet kubeconfig file.
+-->
+11. 完成集群功能的检查之后：
+
+    1. 更新所有的服务账号令牌，使之仅包含新的 CA 证书。
+
+       * 使用使用集群内 kubeconfig 的 Pod 最终也需要被重启，以获得新的服务账号 Secret
```
Comment (SataQiu) [inline content/zh/docs/tasks/tls/manual-rotation-of-ca-certificates.md:270]:
```suggestion
       * 使用集群内 kubeconfig 的 Pod 最终也需要被重启，以获得新的服务账号 Secret
```

**Review by SataQiu — COMMENTED** [review]
Thanks @tengqm 
LGTM other than some nits.

**Review by SataQiu — APPROVED** [review]
/approve
/lgtm

**Pull request merged**